Aftonbladet.js

```diff
@@ -0,0 +1,209 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://ww*\.aftonbladet\.se",
```

Comment:
I use Scaffold, an the first line works, but I can't 'export' that from Scaffold to here, so I have to copy from Scaffold to a text editor.